### PR TITLE
EOS-28075 - cortx-client Helm implementation updates

### PR DESCRIPTION
**Pull request by osowski**
Signed-off-by: Rick Osowski <rosowski@gmail.com>

Based on discussion in EOS-28075 and updates in the `cortx-test` branch here, these changes should be all-encompassing for the cortx-client requirements:
- Adding cortx-client Helm deployments when specified in solution.yaml
- Updating ConfigMaps with appropriate cortx-client endpoints & metadata
- Updating deploy/undeploy scripts for additional elements being deployed/cleaned up.

### CODE REVIEW

File: k8_cortx_cloud/deploy-cortx-cloud.sh [hunk 1]
```diff
@@ -1166,13 +1213,16 @@ function deployCortxClient()
...
             --set cortxclient.sslcfgmap.mountpath="/etc/cortx/solution/ssl" \
+            --set cortxclient.machineid.value="$cortxclient_machineid" \
             --set cortxclient.localpathpvc.name="cortx-client-fs-local-pvc-$node_name" \
             --set cortxclient.localpathpvc.mountpath="$local_storage" \
             --set cortxclient.localpathpvc.requeststoragesize="1Gi" \
             --set namespace=$namespace \
             -n $namespace
     done
 
+exit 1
```
Comment (keithpine) [inline k8_cortx_cloud/deploy-cortx-cloud.sh:1224]: Leftover debug code?

Comment (osowski) [reply k8_cortx_cloud/deploy-cortx-cloud.sh]: D'oh! Good catch. I could have sworn I removed that and it wasn't causing problems in my testing.

**Review by keithpine — APPROVED** [review]
Looks good as far as I can tell. If you want to make Codacy happy, I'd suggest doing that, otherwise, I can fix it later when I address all the others.

**Comment by osowski** [comment]
> Looks good as far as I can tell. If you want to make Codacy happy, I'd suggest doing that, otherwise, I can fix it later when I address all the others.

Since we know a lot of this will be changing soon, I'd rather try to make that happen in one swoop instead of piecing it out across all PRs since that will make our merges that much more difficult.

**Pull request merged**